cloudify_agent/resources/pm/nssm/nssm.conf.template

```diff
@@ -19,7 +19,17 @@ if %errorlevel% neq 0 exit /b %errorlevel%
 
 echo Setting service environment
 
+set "AGENT_PATH={{ virtualenv_path }}\Scripts"
+set NEW_PATH=%AGENT_PATH%;%PATH%
+
+set NEW_PATH=%NEW_PATH:"=%
+set "NEW_PATH1="%NEW_PATH""
+
+echo %NEW_PATH1%
```

Comment:
Add some description to the `echo`
something like `echo New environment path will be %NEW_PATH1%`